lib/ui/painting.dart

```diff
@@ -1606,22 +1606,30 @@ class Codec extends NativeFieldWrapperClass2 {
 /// Instantiates an image codec [Codec] object.
 ///
 /// [list] is the binary image data (e.g a PNG or GIF binary data).
-/// The data can be for either static or animated images.
+/// The data can be for either static or animated images. The following image
+/// formats are supported: {@macro flutter.dart:ui.imageFormats}
 ///
-/// The following image formats are supported: {@macro flutter.dart:ui.imageFormats}
+/// [decodedCacheRatioCap] is the default maximum multiple of the compressed
```

Comment:
Avoid starting with a lower case letter by using a style like `The [foo] parameter is..`